src/objects/npe01__OppPayment__c.object

```diff
@@ -196,6 +210,30 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
+        <fullName>Elevate_Payment_API_Declined_Reason__c</fullName>
+        <description>Provides detail about why a transaction was declined.</description>
```

Comment:
@callen-sfdo Is this a decline reason for the API or for the individual transaction? If so wondering why it's called "API Declined Reason".

Reply:
@Julian88Tex - It's for the individual transaction. We're using "API" in the label to make it clear that it's the value directly from the API (the Elevate backend) not what users see in the front end. At some point later, we may have another field for the declined reason as it's rendered in the UI. 

These API fields are meant to be used for troubleshooting because they're the only way to reconcile the state of records between the two systems in an unambiguous way.